users.js

```diff
@@ -1,6 +1,10 @@
 import React from 'react'
+import MusicianDetails from './musician-details.js'
 
 export default class Users extends React.Component {
+  constructor(props) {
```

Comment:
You don't need this constructor here. In fact, if your component just has a `render` method, it's better to just use a [Stateless Functional Component](https://reactjs.org/docs/components-and-props.html#functional-and-class-components).